src/main/protobuf/v1/party.proto

```diff
@@ -25,10 +25,11 @@ message InstitutionPartyV1 {
   required string description = 3;
   required string digitalAddress = 4;
   optional string code = 5;
-  required string fiscalCode = 6;
-  required string start = 7;
-  optional string end = 8;
-  repeated string attributes = 9;
-  repeated string products = 10;
+  optional string label = 6;
```

Comment:
This field is used for example to store information like ipa category description.
Open to suggestions.

Reply:
Maybe we can make it more specific?
If it is intended for category description we could call it `categoryDescription`?

Reply:
Ok for `categoryDescription`, I was looking for a name not only restricted to the category, for example, in case we were dealing with a private organization (potentially without a category) but maybe, in that case, we can provide a new type of organization.

Reply:
if there is a more generic term for what you have in mind, it's very welcome.
One more point: do we need a description here or we can have a proper identifier?